metapackages/auto-instrumentations-web/package.json

```diff
@@ -33,7 +33,7 @@
     "@opentelemetry/api": "^1.3.0"
   },
   "devDependencies": {
-    "@babel/core": "7.15.0",
+    "@babel/core": "7.22.9",
```

Comment:
seems there is already a new version [7.22.10](https://www.npmjs.com/package/@babel/core/v/7.22.10).

You can optionally bump the version to latest, or we can merge as is and let renovate take care of this